### PR TITLE
testing/wgetpaste: new aport

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: testing/wgetpaste/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Contributor: Rasmus Thomsen <oss@cogitri.dev>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=wgetpaste
+pkgver=2.28
+pkgrel=0
+pkgdesc="Command-line interface to various pastebins"
+url="http://wgetpaste.zlin.dk/"
+arch="noarch"
+license="None"
```
Comment (maxice8) [inline testing/wgetpaste/APKBUILD]: Public Domain

Comment (Cogitri) [reply testing/wgetpaste/APKBUILD]: But that's not listed in SPDX either and there's no license file attached :c

Comment (maxice8) [reply testing/wgetpaste/APKBUILD]: i think it should be fine, public domain isn't a license anyways

Comment (Cogitri) [reply testing/wgetpaste/APKBUILD]: I'm currently working with upstream to get this fixed, I'll push again then.

**Pull request closed without merging**